docs/pricing.md

```diff
@@ -44,43 +44,35 @@ outer component styling (optional)
 | :------------: | :---------------: |
 | object (style) | inherited styling |
 
-### `buttonFont`
-
-specify button font family
-
-|  Type  |                 Default                 |
-| :----: | :-------------------------------------: |
-| string | System font (iOS), Sans Serif (android) |
-
 ---
 
-### `infoFont`
+### `infoStyle`
 
-specify pricing information font family
+specify pricing information style
 
 |  Type  |                      Default                      |
 | :----: | :-----------------------------------------------: |
-| string | System font bold (iOS), Sans Serif Bold (android) |
+| object |                       none                        |
```

Comment:
Let's use `object (style)` like the other pages. e.g https://react-native-training.github.io/react-native-elements/docs/searchbar.html#inputcontainerstyle

Reply:
sorry for the delay man.